demo/firebase-messaging.html

```diff
@@ -60,6 +60,9 @@
       window.addEventListener('WebComponentsReady', function() {
         if (navigator.serviceWorker) {
           navigator.serviceWorker.register('firebase-messaging-sw.js').then(function(sw) {
+            var app = document.getElementById('app');
+            // polymer 2.0 compatibility
+            app = app.$ ? app : app.parentNode;
```

Comment:
well in Polymer 1 `<dom-bind>` wont have `.$` and in Polymer 2 `<template is="dom-bind"` won't have `.$`